### PR TITLE
cors setup and whitelisting 

### PR DESCRIPTION
**Pull request by khushi818**

### CODE REVIEW

File: django_jsonapi/settings/base.py [hunk 1]
```diff
@@ -179,3 +181,8 @@
     ),
     'TEST_REQUEST_DEFAULT_FORMAT': 'vnd.api+json'
 }
+
+CORS_ALLOWED_ORIGINS_REGEXES = [
```
Comment (pallabez) [inline django_jsonapi/settings/base.py]:
```suggestion
CORS_ALLOWED_ORIGIN_REGEXES = [
```

**Comment by pallabez** [comment]
Thanks for the PR. Can you write test for it as well in a different PR?

We need to check if the regex pattern matches `https://dev.realdevsquad.com` & `http://localhost:3000` in tests.

**Comment by khushi818** [comment]
> Thanks for the PR. Can you write test for it as well in a different PR?
> 
> We need to check if the regex pattern matches `https://dev.realdevsquad.com` & `http://localhost:3000` in tests.

Sure i will write it

File: requirements.txt [hunk 2]
```diff
@@ -1,16 +1,20 @@
...
 inflection==0.5.1
 pycodestyle==2.9.1
+python-dotenv==0.21.0
 pytz==2022.2.1
+sentry-sdk==1.22.2
 sqlparse==0.4.2
 toml==0.10.2
-gunicorn==20.1.0
+tzdata==2023.3
+urllib3==1.26.15
```
Comment (pallabez) [inline requirements.txt:19]: Are we sure we need these?

Comment (khushi818) [reply requirements.txt:19]: yes I am sure we need these. we have 3 extra libraries certifi , tzdata, urllib3 which came with other packages. All are useful I searched google

**Pull request merged**